recipes/bzip2/1.0.8/test_package/conanfile.py

```diff
@@ -4,15 +4,16 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "arch", "build_type"
-    generators = "cmake"
+    generators = "cmake", "pkg_config"
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
-        
+
     def test(self):
         assert os.path.isfile(os.path.join(self.deps_cpp_info["bzip2"].rootpath, "licenses", "LICENSE"))
+        assert os.path.isfile(os.path.join(self.build_folder, "bzip2.pc"))
```

Comment:
it's not required because you are testing a Conan feature, not a package artifact. Anyway, this test does not hurt, but avoid it in the future.